udata_front/views/gouvfr.py

```diff
@@ -111,17 +111,40 @@ def get_object(model, id_or_slug):
     return obj
 
 
+def get_objects_from_tag(model, tag) -> list:
+    objects = getattr(model, "objects").filter(tags=tag)
+    return [r for r in objects]
+
+
 @blueprint.route('/pages/<path:slug>/')
 def show_page(slug):
     content, gh_url, extension = get_page_content(slug)
     page = frontmatter.loads(content)
-    reuses = [get_object(Reuse, r) for r in page.get('reuses') or []]
-    datasets = [get_object(Dataset, d) for d in page.get('datasets') or []]
-    reuses = [r for r in reuses if r is not None]
-    datasets = [d for d in datasets if d is not None]
+
+    data = {}
+    models = {"reuses": Reuse, "datasets": Dataset}
+
+    for data_type, model in models.items():
```

Comment:
I wonder if we could have a get_objects function that encapsulate this logic? For readability of the `show_page` function.